homeassistant/components/sense/strings.json

```diff
@@ -49,6 +49,123 @@
       },
       "bill_energy": {
         "name": "Bill energy"
+      },
+      "voltage_0": {
+        "name": "L1 voltage"
+      },
+      "voltage_1": {
+        "name": "L2 voltage"
+      },
+      "active_usage": {
+        "name": "Energy"
+      },
+      "active_production": {
+        "name": "Production"
+      },
+      "daily_production": {
+        "name": "Daily production"
+      },
+      "weekly_production": {
+        "name": "Weekly production"
+      },
+      "monthly_production": {
+        "name": "Monthly production"
+      },
+      "yearly_production": {
+        "name": "Yearly production"
+      },
+      "bill_production": {
+        "name": "Bill production"
+      },
+      "daily_usage": {
+        "name": "Daily energy"
+      },
+      "weekly_usage": {
+        "name": "Weekly energy"
+      },
+      "monthly_usage": {
+        "name": "Monthly energy"
+      },
+      "yearly_usage": {
+        "name": "Yearly energy"
+      },
+      "bill_usage": {
+        "name": "Bill energy"
+      },
```

Comment:
Are there also entities that we automatically disable by default?

Reply:
Yes, the percentage ones were disabled by default, fixed that here